src/RpcServer/RpcServer.SmartContract.cs

```diff
@@ -271,7 +271,7 @@ protected virtual JToken GetUnclaimedGas(JArray _params)
                 script_hash = null;
             }
             if (script_hash == null)
-                throw new RpcException(-100, "Invalid address");
+                throw new RpcException(RpcError.InvalidParams);
```

Comment:
L255-256 in this file: we need to come to agreement about behaviour. If session is not found on `terminatesession` request, then NeoC# doesn't throw an exception and returns `false`. NeoGo returns `UnknownSession` RPC error. What is the desired behaviour?

@shargon, @roman-khimov

Reply:
Reminds me of #745. But "Unknown session" is exactly for that, so we better use it.